internal/kafka/command_partition_describe_onprem.go

```diff
@@ -0,0 +1,64 @@
+package kafka
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafkarest"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *partitionCommand) newDescribeCommandOnPrem() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe <id>",
+		Short: "Describe a Kafka partition.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.describeOnPrem,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Describe partition "1" for topic "my_topic".`,
+				Code: "confluent kafka partition describe 1 --topic my_topic",
+			},
+		),
+	}
+
+	cmd.Flags().String("topic", "", "Topic name to list partitions of.")
```

Comment:
Is this description correct? I thought we pass the partition number to this command.